Sources/UserScript/UserScriptMessageEncryption.swift

```diff
@@ -78,4 +78,8 @@ extension UserScriptMessageEncryption {
             message.messageWebView?.evaluateJavaScript(script)
         }
     }
+
+    public func messageHandlerFor(_ messageName: String) -> MessageHandler? {
```

Comment:
it should be public func messageHandler(for messageName ...

Reply:
This one already existed and is used by Autofill etc :)